pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -193,14 +198,24 @@ func gatherNamespaceEvents(ctx context.Context, coreClient corev1client.CoreV1In
 		return nil, err
 	}
 	// filter the event list to only recent events
-	oldestEventTime := time.Now().Add(-maxEventTimeInterval * 2)
+	oldestEventTime := time.Now().Add(-interval)
```

Comment:
So here we go back to collecting the data within the normal range. Is this intentional?

Reply:
Yes I think it makes sense. If you change the default conf and the gathering runs e.g every 4 hours then you will be interested in the events from the last 4 hours (and not only 2).